src/driver.h

```diff
@@ -25,7 +25,7 @@ class Driver {
 	 * @param parm Parameters passed to the driver.
 	 * @return nullptr if everything went okay, otherwise an error message.
```

Comment:
This comment is now out of date.

I'm wondering whether we want to use "the string is empty" as "there is no error". Might wrapping the string view in `std::optional` be a clearer API, as the existence of the string means an error?

Reply:
I too would prefer `std::optional` here.